Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -64,12 +87,17 @@ static private void initHttpClient()
                 // Enforce tls v1.2
                 SslProtocols = SslProtocols.Tls12,
                 AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate,
-            };
+                CookieContainer = cookieContainer = new CookieContainer()

```

Comment:
Why do we need this?

Reply:
umh, so I looked back into the file history. It seems that I inadvertently removed it last time and none of use caught it. This is necessary for subsequent Okta connection to work (https://github.com/snowflakedb/snowflake-connector-net/pull/272).